cockroachdb-in-comparison.md

```diff
@@ -2,3 +2,19 @@
 title: CockroachDB in Comparison
 toc: false
 ---
+
+| | CockroachDB | MySQL | PostgreSQL | Cassandra | HBase | MongoDB | Riak 
+--|--|--|--|--|--|--|--
+Automated Scaling | Yes | No | No | Yes | Yes  | Yes | Yes 
+Automated Failover & Repair | Yes | No | No | Yes | Yes | Yes | Yes 
```

Comment:
It might make sense to reword it as

- eventual consistent read from any replica

to which our answer would be no for now.


On Wed, Feb 17, 2016 at 11:08 AM Spencer Kimball <notifications@github.com>
wrote:

> In cockroachdb-in-comparison.md
> <https://github.com/cockroachdb/docs/pull/52#discussion_r53186478>:
>
> > @@ -2,3 +2,19 @@
> >  title: CockroachDB in Comparison
> >  toc: false
> >  ---
> > +
> > +| | CockroachDB | MySQL | PostgreSQL | Cassandra | HBase | MongoDB | Riak
> > +--|--|--|--|--|--|--|--
> > +Automated Scaling | Yes | No | No | Yes | Yes  | Yes | Yes
> > +Automated Failover & Repair | Yes | No | No | Yes | Yes | Yes | Yes
>
> It isn't currently possible through SQL, though that would be relatively
> easy to change. For now we can go ahead and say "No" for that feature I
> suppose.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/docs/pull/52/files#r53186478>.
>